builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -351,13 +357,14 @@ function store.get_formspec()
 	end
 
 	local formspec = {
-		"size[12,6.5;true]",
-		"field[0.3,0.1;10.2,1;search_string;;", core.formspec_escape(search_string), "]",
+		"size[12,7;true]",
+		"position[0.5,0.55]",
+		"field[0.2,0.1;7.8,1;search_string;;", core.formspec_escape(search_string), "]",
 		"field_close_on_enter[search_string;false]",
-		"button[10.2,-0.2;2,1;search;", fgettext("Search"), "]",
-		"dropdown[0,1;2.4;type;",
+		"dropdown[7.7,-0.1;2.4;type;",
 		table.concat(filter_types_titles, ","),
-		";",
+		";]",
+		"button[10.1,-0.2;2,1;search;", fgettext("Search"), "]",
 		filter_type,
```

Comment:
In wrong place? Try moving to just after
```
table.concat(filter_types_titles, ","),
";",
```

Reply:
Yes seems to work:
```
	local formspec = {
		"size[12,7;true]",
		"position[0.5,0.55]",
		"field[0.2,0.1;7.8,1;search_string;;", core.formspec_escape(search_string), "]",
		"field_close_on_enter[search_string;false]",
		"dropdown[7.7,-0.1;2.4;type;",
		table.concat(filter_types_titles, ","),
		";",
		filter_type,
		"]",
		"button[10.1,-0.2;2,1;search;", fgettext("Search"), "]",
		-- "textlist[0,1;2.4,5.6;a;",
		-- table.concat(taglist, ","),
		-- "]"
	}
```

Reply:
Not sure how that happened, will fix.